cgmanifest.json

```diff
@@ -29763,7 +29923,7 @@
         "other": {
           "name": "xerces-c",
           "version": "3.2.4",
-          "downloadUrl": "https://downloads.apache.org/xerces/c/3/sources/xerces-c-3.2.4.tar.xz"
+          "downloadUrl": "https://downloads.apache.org/xerces/c/3/sources/xerces-c-3.2.4.tar.gz"
```

Comment:
I think it should be `xz` as per the [spec](https://github.com/microsoft/CBL-Mariner/blob/main/SPECS/xerces-c/xerces-c.spec#L9)